euphoria-service/src/test/java/edu/cooper/ece366/euphoria/ApplicationHandlersTest.java

```diff
@@ -32,6 +35,8 @@
     PreparedStatement ps;
     @Mock
     ResultSet rs;
+    @Mock
+    byte[] requestBytes;
```

Comment:
I'm pretty sure you can only mock Java objects, not primitives. However, you probably shouldn't mock your request payload because you want to verify that for a given request input you get the right output. 

Something like 

```java
when(mockRequestContext.request()).thenReturn(mockRequest);
when(mockRequest.payload()).thenReturn(Optional.of(requestPayloadByteString));
```

Reply:
Then in your test code you would also have

```java
when(mockObjectMapper.readValue(requestPayloadByteString, Book.class)).thenReturn(book);
```

Reply:
I'm not sure I understand - what is `requestPayloadByteString` here? Is it a mocked variable?

Reply:
Never mind, figured it out. Some tests are still failing, but that's for a separate reason. Thanks!